static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -120,11 +120,13 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
           </Blockquote>
         </Section>
 
-        <Section icon={<IconLink size="xs" />} title={t('Url')}>
-          <ErrorBoundary mini>
-            <TextCopyInput size="sm">{url?.value ?? t('URL not found')}</TextCopyInput>
-          </ErrorBoundary>
-        </Section>
+        {url && (
```

Comment:
Hmm, let's keep the section header if it's loading, so that we have less DOM elements jumping around when it does load. I would either placeholder over TextCopyInput or show empty string in its value while it's loading instead of hiding the entire section

Reply:
updated!